src/timepicker-select/timepicker-select.stories.ts

```diff
@@ -0,0 +1,36 @@
+import { TimePickerSelectModule } from "./timepicker-select.module";
+import { storiesOf, moduleMetadata } from "@storybook/angular";
+import { withKnobs } from "@storybook/addon-knobs/angular";
+import { ExperimentalComponenent } from "../../.storybook/experimental.component";
+import { ExperimentalModule } from "..";
+
+storiesOf("Time Picker Select", module)
+	.addDecorator(
+		moduleMetadata({
+			declarations: [ExperimentalComponenent],
+			imports: [
+				TimePickerSelectModule,
+				ExperimentalModule
+			]
+		})
+	)
+	.addDecorator(withKnobs)
+	.add("Simple", () => ({
+		template: `
+		<app-experimental-component></app-experimental-component>
+		<div class="bx--form-item">
+			<div class="bx--time-picker">
+				<ibm-timepicker-select>
+					<option value="" disabled selected hidden>AM</option>
```

Comment:
You probably want to have the actually selected option be `selected` ... then you can drop this weird option that's disabled, selected, hidden

Reply:
(this applies everywhere this pattern is used)